lib/bench/either.bench.js

```diff
@@ -1,4 +1,4 @@
-var R = require('../..');
+var R = require('../../source');
```

Comment:
We have two separate ways we're requiring the function in these tests, and we should standardize on one if we're gonna go in and change them all anyways. I _think_ I prefer `var f = require('../../source').f` over this alternative, but I would defer to others if they have a different opinion because I don't have a strong preference.

Reply:
I prefer `var f = require(../../source/f)`.